syntax/verilog_systemverilog.vim

```diff
@@ -187,8 +187,7 @@ for name in ['class', 'clocking', 'covergroup', 'function', 'interface',
     endif
 
     if verilog_systemverilog#VariableExists('verilog_quick_syntax')
-        execute 'syn match verilogStatement "'.s:region_start.'"'
-        execute 'syn match verilogStatement "'.s:region_end.'"'
+        execute 'syn keyword verilogStatement '.s:region_start.' '.s:region_end
```

Comment:
Does this work? The reason we used 'syn-match' was for the cases where regions had de-qualifiers, e.g. `typedef class`